src/AST-Core-Traits/TRBProgramNodeVisitor.trait.st

```diff
@@ -128,6 +128,11 @@ TRBProgramNodeVisitor >> visitSuperNode: aSuperNode [
 	^ self visitVariableNode: aSuperNode
 ]
 
+{ #category : #visiting }
+TRBProgramNodeVisitor >> visitTemporaryDeclarationNode: aTemporaryDeclarationNode [
+	^self visitArgumentNode: aTemporaryDeclarationNode 
```

Comment:
can you add a comment stating that this is a backward compatible solution because this is strange that visitArgumentNode: visit a temp.

Reply:
Took the one I put in RBVisitor, as suggested in the your last comment